### PR TITLE
fix error on undefined plugin

### PR DESCRIPTION
**Pull request by dancastillo**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

fixes: https://github.com/fastify/fastify-cli/issues/633

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

File: test/start.test.js [hunk 1]
```diff
@@ -1034,3 +1034,24 @@ test('should start fastify with custom plugin options with a CJS plugin with pac
...
+        t.equal(err.code, 'AVV_ERR_PLUGIN_NOT_VALID')
+      }
+    }
+  })
+
+  const port = getPort()
+
+  try {
+    const argv = ['-p', port, '-T', '100', './test/data/object.js']
+    await start.start(argv)
```
Comment (Eomm) [inline test/start.test.js:1053]:
```suggestion
    await start.start(argv)
    t.fail('should not start')
```

File: test/start.test.js [hunk 2]
```diff
@@ -1034,3 +1034,24 @@ test('should start fastify with custom plugin options with a CJS plugin with pac
   t.pass('server closed')
   t.end()
 })
+
+test('should throw error for invalid fastify plugin (object)', async t => {
+  t.plan(1)
```
Comment (Eomm) [inline test/start.test.js:1039]: I count 2 assertion but only 1 is planned - so one of the two is not executed

Comment (dancastillo) [reply test/start.test.js:1039]: Should be updated now @Eomm

**Review by Eomm — APPROVED** [review]

**Pull request merged**